docs/en_US/NAS/Benchmarks.md

```diff
@@ -158,3 +160,22 @@ Here is a list of available operators used in NDS.
 
 .. autoclass:: nni.nas.benchmarks.nds.NdsIntermediateStats
 ```
+
+## NLP
+
+[Paper link](https://arxiv.org/abs/2006.07116) &nbsp; &nbsp; [Open-source](https://github.com/fmsnew/nas-bench-nlp-release)
```

Comment:
have you verified that the accuracy number can be reproduced with the authors' code?

Reply:
Not really. Do I need to double check that? If so, I will try to run the paper's repo's code.